packages/modules/imodel-browser/src/containers/iModelTiles/IModelTile.tsx

```diff
@@ -50,25 +50,26 @@ export const IModelTile = ({
         }
       : undefined;
   return (
-    <Tile
-      key={iModel?.id}
-      name={<span title={iModel?.displayName}>{iModel?.displayName}</span>}
-      description={
-        <span title={iModel?.description ?? ""}>
-          {iModel?.description ?? ""}
-        </span>
-      }
-      moreOptions={moreOptions}
-      thumbnail={
-        <IModelThumbnail
-          className={"iui-picture"}
-          iModelId={iModel?.id}
-          accessToken={accessToken}
-          onClick={() => onThumbnailClick?.(iModel)}
-          apiOverrides={thumbnailApiOverride}
-        />
-      }
-      {...(tileProps ?? {})}
-    />
+    <ThemeProvider theme="inherit">
```

Comment:
all looks good.
last question, tiles can be used outside of grid?

Reply:
Yes

Reply:
ok.. weird...